src/test/java/org/broadinstitute/hellbender/utils/test/FuncotatorTestUtils.java

```diff
@@ -49,4 +59,114 @@ public static FeatureContext createFeatureContext(final List<DataSourceFuncotati
         return FeatureContext.createFeatureContextForTesting(featureInputsWithType, dummyToolInstanceName, interval,
                 featureQueryLookahead, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, reference);
     }
+
+    /**
+     * Create a dummy gencode funcotation in the most manual way possible.  This method does not check for consistency
+     *  or reasonable values.
+     *
+     * @param hugoSymbol
```

Comment:
It's kind of a pain, but can you fill in some descriptions for these fields?

Reply:
Done.